go/stellar/stellar.go

```diff
@@ -294,14 +309,21 @@ type DisplayBalance struct {
 // User with wallet ready : Standard payment
 // User without a wallet  : Relay payment
 // Unresolved assertion   : Relay payment
-func SendPayment(m libkb.MetaContext, remoter remote.Remoter, to stellarcommon.RecipientInput, amount string, note string, displayBalance DisplayBalance, forceRelay, quickReturn bool, publicNote string) (res stellar1.SendResultCLILocal, err error) {
+func SendPayment(m libkb.MetaContext, remoter remote.Remoter, to stellarcommon.RecipientInput, amount string, note string, displayBalance DisplayBalance, forceRelay, quickReturn bool, publicNote string, fromAccountID stellar1.AccountID) (res stellar1.SendResultCLILocal, err error) {
```

Comment:
Move `fromAccountID` to before `to`. Could just be `from` since it's typed. Also comment that it's optional.